src/style.css

```diff
@@ -0,0 +1,73 @@
+body {
+  display: flex;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  margin: 0;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+.block {
+  color: white;
+  display: flex;
+  height: 300px;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  min-width: 300px;
+  flex-basis: 26%;
```

Comment:
It looks like a magic number: `26%`. Find another solution to avoid using this magic number or create a `css` variable.